src/error-index-generator/main.rs

```diff
@@ -73,7 +73,7 @@ r##"<!DOCTYPE html>
 
     try!(write!(&mut output_file, "<h1>Rust Compiler Error Index</h1>\n"));
 
-    for (err_code, info) in err_map.iter() {
+    for (err_code, info) in err_map {
```

Comment:
IMHO, this isn't particularly worrisome because the default - move if it's a value or take references if it's a reference - is always either the right thing or a compiler error. YMMV.